paddle/fluid/imperative/partial_grad_engine.cc

```diff
@@ -216,7 +228,10 @@ static void GetGraphInfoBetweenTargets(
     auto iter = preceding_ops.find(op);
     if (iter != preceding_ops.end()) {
       for (auto &preceding_op : iter->second) {
-        op_queue.emplace(preceding_op, op);
+        if (op_base_visited.count(std::make_pair(preceding_op, op)) == 0) {
+          op_queue.emplace(preceding_op, op);
+          op_base_visited.insert(std::make_pair(preceding_op, op));
```

Comment:
Prefer to use `op_base_visited.emplace(preceding_op, op);` for the conciseness and the performance. This is just a suggestion.

Reply:
DONE